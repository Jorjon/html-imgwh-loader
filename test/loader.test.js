import loader from '../';

describe('Loader', () => {
    let thisObject = {
        resourcePath: __filename,
    };
    test('image.png should have proper width/height attributes', () => {
        expect(loader.call(thisObject, "<img src=\"image.png\"/>")).toMatchSnapshot();
    });
    test('should not add width if it is already present', () => {
        expect(loader.call(thisObject, "<img src=\"image.png\" width=\"50\"/>")).toMatchSnapshot();
    });
    test('should not add height if it is already present', () => {
        expect(loader.call(thisObject, "<img src=\"image.png\" height=\"50\"/>")).toMatchSnapshot();
    });
});