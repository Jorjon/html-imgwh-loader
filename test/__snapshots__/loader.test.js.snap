// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader image.png should have proper width/height attributes 1`] = `"<img width=\\"240\\" height=\\"240\\" src=\\"image.png\\"/>"`;

exports[`Loader should not add height if it is already present 1`] = `"<img width=\\"240\\" src=\\"image.png\\" height=\\"50\\"/>"`;

exports[`Loader should not add width if it is already present 1`] = `"<img height=\\"240\\" src=\\"image.png\\" width=\\"50\\"/>"`;
